From 3edf5816a31f68df6aafe53f6cd34180ee0f4e3c Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Fri, 21 May 2021 23:02:23 +0200
Subject: [PATCH 14/N] Remove fakeroot

Currently we don't have fakeroot, so let's allow building without it.  Most of
the patching is currently done in 0000-pacman-msysize.patch, these are just some
new occurrences.  Ideally this removal will be a completely separate patch in
the future.
---
 scripts/libmakepkg/executable/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/scripts/libmakepkg/executable/meson.build b/scripts/libmakepkg/executable/meson.build
index 8536a54..d5b6dcc 100644
--- a/scripts/libmakepkg/executable/meson.build
+++ b/scripts/libmakepkg/executable/meson.build
@@ -4,7 +4,6 @@ sources = [
   'ccache.sh.in',
   'checksum.sh.in',
   'distcc.sh.in',
-  'fakeroot.sh.in',
   'gpg.sh.in',
   'gzip.sh.in',
   'pacman.sh.in',
-- 
2.31.1

